lazy_tensor_core/lazy_bench.py

```diff
@@ -304,7 +307,11 @@ def lazy_overhead_experiment(args, results, benchmark, lazy_benchmark):
     for rep in range(args.repeat):
         # interleave the runs to handle frequency scaling and load changes
         _, timings[rep, 0] = timed(args, benchmark, sync=ref_sync(sync_every_iter=True))
-        _, timings[rep, 1] = timed(args, lazy_benchmark, sync=NoOpSync())
+        _, timings[rep, 1] = timed(args, lazy_benchmark, sync=LazySync(skip_final_sync=True))
```

Comment:
Were we measuring the execution time in addition to tracing before? Or, in other words, what would be the effect of this change on `timings`?

Reply:
Previously, we would have measured a less deterministic quantity.  We were still measuring lazy tracing, but we were never executing the part of the code that looks up the hash in the cache and submits an async execution task.  In this case it should just be a little more deterministic and faithful to what we are trying to measure.

Reply:
subtle :)